pkg/deploy/deploy.go

```diff
@@ -122,6 +129,18 @@ func Deploy(ctx context.Context, projects []project.Project, environmentClients
 	return nil
 }
 
+func getSortedEnvConfigs(g graph.ConfigGraphPerEnvironment, environmentClients dynatrace.EnvironmentClients) (map[string][]graph.SortedComponent, error) {
```

Comment:
This function could also just take the environment names, obtained from `environmentClients.Names()`, similar to `graph.New(...)`.

Reply:
Or it could even move to the `graph` package or `ConfigGraphPerEnvironment`type?